styles/mixins.scss

```diff
@@ -63,92 +65,94 @@
     }
 }
 
+$-pin-mixin-variables-ns: --_--g-pin-mixin-border-radius;
+
+@mixin -pin-mixin-variables-from-list($list) {
+    $props: top-left, top-right, bottom-right, bottom-left;
+    $indexes: 1, 2, 3, 4;
+
+    @if list.length($list) == 1 {
+        $indexes: 1, 1, 1, 1;
+    }
+
+    @each $prop, $index in list.zip($props, $indexes) {
+        #{#{$-pin-mixin-variables-ns}-#{$prop}}: list.nth($list, $index);
+    }
+}
+
 @mixin pin($block, $selectors, $radius, $append: true) {
+    @each $selector in $selectors {
+        &#{get-pin-selector($block, $selector, $append)} {
+            border-radius: (
+                var(#{$-pin-mixin-variables-ns}-top-left)
+                    var(#{$-pin-mixin-variables-ns}-top-right)
+                    var(#{$-pin-mixin-variables-ns}-bottom-right)
+                    var(#{$-pin-mixin-variables-ns}-bottom-left)
```

Comment:
RTL support degradation

Reply:
Why?

Reply:
physical properties instead of logical ones

Reply:
But property is `border-radius`, it is rtl compatible

Reply:
There should be not such api like `top-left`, `bottom-right`. Instead we should only use logical names: `start-start`, `end-end`, etc.

Reply:
Oh, i will rename vars, no problem